logging/telemetryhook.go

```diff
@@ -199,6 +202,24 @@ func (hook *dummyHook) waitForEventAndReady() bool {
 	return true
 }
 
+type elasticClientLogger struct {
+	elastic.Logger
```

Comment:
to be a elastic logger just needs Printf, which always points to Base(), so I think we don't need this part of the struct.